src/Nethermind/Ethereum.Test.Base/BlockchainTestBase.cs

```diff
@@ -150,12 +151,14 @@ protected async Task<EthereumTestResult> RunTest(BlockchainTest test, Stopwatch?
                 specProvider,
                 blockValidator,
                 rewardCalculator,
-                new TransactionProcessor(
-                    specProvider,
-                    stateProvider,
-                    storageProvider,
-                    virtualMachine,
-                    _logManager),
+                new BlockProcessor.ProcessBlockTransactionsStrategy(
```

Comment:
strange class name -> what strategy?
sounds like a general name is there but not specific strategy?

Reply:
Strategy to process block transactions of course, contrary to  the strategy to produce block. I'm open for better naming.

Reply:
Strategy to process block transactions of course, contrary to  the strategy to produce block. I'm open for better naming.